Include/GWCA/Packets/StoC.h

```diff
@@ -102,6 +102,14 @@ namespace GW {
             };
             const uint32_t Packet<MessageServer>::STATIC_HEADER = 0x5E;
 
+            // Deliver chat message (sender is an NPC)
+            struct MessageNPC : Packet<MessageNPC> {
+                uint32_t agent_id;
+                uint32_t type;
+                wchar_t sender_name[32];
```

Comment:
Are you sure it's `wchar_t[32]` ? I have `wchar_t[8]`.

Reply:
No I'm not sure, but you're probably right as its a hash anyway. Any way to log the capacity of the strings in your logger, not just the size?

Reply:
Sure, but the logger did print something lower than 8, right ?

Reply:
Yeh I got a string length of 4 chars consistently when testing in a few factions missions but didn't know if the hash could be longer for other NPCs